python/ray/data/read_api.py

```diff
@@ -172,9 +183,18 @@ def range_tensor(
 
     Examples:
         >>> import ray
-        >>> ds = ray.data.range_tensor(1000, shape=(3, 10)) # doctest: +SKIP
-        >>> ds.map_batches( # doctest: +SKIP
-        ...     lambda arr: arr * 2).show()
+        >>> ds = ray.data.range_tensor(1000, shape=(2, 2)) # doctest: +SKIP
+        >>> ds # doctest: +SKIP
+        Dataset(
+            num_blocks=200,
+            num_rows=1000,
+            schema={__value__: <ArrowTensorType: shape=(2, 2), dtype=int64>},
+        )
+        >>> ds.map_batches(lambda arr: arr * 2).take(2) # doctest: +SKIP
+        [array([[0, 0],
+                [0, 0]]),
+        array([[2, 2],
```

Comment:
nit: indent

Reply:
Hmm this is the exact output, what's the nit about indent? Just prettifying it?